doc/modules/tree.rst

```diff
@@ -126,41 +126,35 @@ Using the Iris dataset, we can construct a tree as follows::
 
 Once trained, we can export the tree in `Graphviz
 <http://www.graphviz.org/>`_ format using the :func:`export_graphviz`
-exporter. Below is an example export of a tree trained on the entire
-iris dataset::
+exporter. If you use the `conda<http://conda.io>`_ package manager, the graphviz binaries  
+and the python package can be installed with 
 
-    >>> with open("iris.dot", 'w') as f:
-    ...     f = tree.export_graphviz(clf, out_file=f)
+    conda install graphviz
+    pip install graphviz 
```

Comment:
this will not work because of the way conda installs graphviz. You need to ``conda install python-graphviz`` not ``pip install graphviz``.